### DISCUSSION TITLE
posts/fast-internet-test-cli/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#225] [Comments]
# posts/fast-internet-test-cli/

What is FAST.com?

https://technotim.live/posts/fast-internet-test-cli/

**Comment by dpearceMN** [comment]
If all you want is the final number:
```bash
./fast | tr '\r' '\n' | tail -n 1 | cut -d ' ' -f 3
```

**Comment by jacobslusser** [comment]
This sorta hints at a question I've had about your home lab. What is your home internet speed? This suggests maybe 500/500?

**Reply by timothystewart6** [reply]
That's right, it's currently 500/500 which is way more than I need but it's the cheapest plan!

**Comment by LaM4PL** [comment]
Hi :)
There is still curl in wget version ;)  It could be like this:
```
LATEST_VERSION=$(wget -q -O-  "https://api.github.com/repos/ddo/fast/releases/latest" | grep -Po '"tag_name": "v\K[0-9.]+')
```

Great lab and tutorials btw :)
